packages/app-mobile/components/plugins/backgroundPage/pluginRunnerBackgroundPage.ts

```diff
@@ -8,7 +8,7 @@ const legacyPluginIds = [
 ];
 
 const pathLibrary = require('path');
-const punycode = require('punycode/');
```

Comment:
Be aware that this imports `punycode.js` and not the `punycode` built-in (this runs in an `<iframe>` on mobile where the built-in NodeJS `punycode` is unavailable).